src/components/CountrySelector/helpers.ts

```diff
@@ -0,0 +1,11 @@
+import { Country } from "types/components";
+import countries from "assets/countries/all.json";
+
+export const country = (name: string): Country => {
```

Comment:
```suggestion
export const country = (name = ""): Country => {
```
nit: makes it unnecessary to do `country(props.hq_country ?? "")`